nvdaHelper/interfaces/nvdaController/nvdaController.idl

```diff
@@ -26,12 +26,39 @@ cpp_quote("This license can be found at:")
 cpp_quote("http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html")
 cpp_quote("*/")
 
+typedef enum {

```

Comment:
could you add a comment on this Enum that it must match the enum in python code.
Same with in the python definition, so that these 2 enums are kept in sync.

Reply:
It seems like these comments are missing from the python module